scheds/rust/scx_layered/src/main.rs

```diff
@@ -1935,7 +1935,10 @@ impl<'a> Scheduler<'a> {
             return Ok(());
         }
 
-        let nvml = NVML_CELL.get_or_try_init(|| Nvml::init())?;
+        let nvml = NVML_CELL.get_or_try_init(|| {
+            Nvml::init().context(format!("enabling GPU support requires a nvidia device"))
```

Comment:
```suggestion
            Nvml::init().context("enabling GPU support requires a nvidia device")
```

What's the `format` for?

Reply:
nothing.. was trying something else and forgot to remove it.